test/integration-tests/MessageDeadlineCheck.t.sol

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.9;
+
+/// library imports
+import {Vm} from "forge-std/Test.sol";
+
+/// local imports
+import "test/Setup.t.sol";
+
+import {MultiMessageSender} from "src/MultiMessageSender.sol";
+import {MultiMessageReceiver} from "src/MultiMessageReceiver.sol";
+import {Error} from "src/libraries/Error.sol";
+import {GovernanceTimelock} from "src/controllers/GovernanceTimelock.sol";
+
+/// @dev scenario: demonstrate that a admin-specified deadline is good
```

Comment:
is this comment still valid?

Reply:
Is this test scenario still useful, given the changes made to message expiration validation rules